modules/voting/admin/del.php

```diff
@@ -27,6 +27,15 @@
         $db->query("DELETE FROM " . NV_PREFIXLANG . "_" . $module_data . "_rows WHERE vid=" . $vid);
         $nv_Cache->delMod($module_name);
 
+        $dir = NV_ROOTDIR . '/' . NV_LOGS_DIR . '/voting_logs';
+        $pattern = '/^vo' . $vid . '/';
```

Comment:
@baonv-dev `$pattern` này không an toàn em, phải chỉ ra thêm ký tự _. Ví dụ anh có ID = 10 và ID = 100 nếu anh xóa cái ID = 10 đi thì với cái `$pattern` của em ID = 100 cũng bị xóa mất